libs/designsystem/ng-package.json

```diff
@@ -18,6 +18,7 @@
     "highcharts",
     "moment",
     "@fontsource/roboto",
-    "inputmask"
+    "inputmask",
+    "chart.js"
```

Comment:
Do you also need to add this to `umdModuleIds`? Do you get a warning about that when publishing? 🤔